jni/src/main/java/com/horizen/schnorrnative/SchnorrKeysList.java

```diff
@@ -0,0 +1,91 @@
+package com.horizen.schnorrnative;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.horizen.librustsidechains.Library;
+import com.horizen.librustsidechains.FieldElement;
+
+public class SchnorrKeysList implements AutoCloseable {
+    private SchnorrPublicKey[] signingKeys;
+    private SchnorrPublicKey[] masterKeys;
+    private SchnorrPublicKey[] updatedSigningKeys;
+    private SchnorrPublicKey[] updatedMasterKeys;
```

Comment:
We need to agree on the format of these: I guess these should contain the same keys as the old ones, if they were not updated, and the new ones if they were updated. But it might be easier for the SDK to express this in another way. @sgonta had some ideas. @i-Alex